### PR TITLE
feat: catppuccin colorscheme package

### PR DESCRIPTION
**Pull request by ghostx31**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **
- [x] Any commands are available via the command palette.
- [x] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.

My package is ... Catppuccin colourscheme. It can be viewed at [catppuccin/sublime-text](https://github.com/catppuccin/sublime-text).

There are no packages like it in Package Control.

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

**Comment by stfnx** [comment]
This one is almost two months old now... when will it get merged?

**Comment by ghostx31** [comment]
We are also waiting for it to get merged. Don't know if I should ping the maintainer to have a look at this and get this merged.

File: repository/c.json
Comment (braver) [inline repository/c.json:421]: Please add the color-scheme label.

File: repository/c.json
Comment (braver) [inline repository/c.json:419]: Typically it's a good idea to add "color scheme", or in this case "colors schemes" in the name.

**Comment by braver** [comment]
I think the assets directory and probably also the python file should be excluded from the package using `.gitattributes` file.

> Soothing pastel theme for Sublime Text

In Sublime Text parlance your package isn't a theme, it's a (collection of) color scheme(s). So I would suggest using the correct wording here also.

**Comment by ghostx31** [comment]
I've removed the `assets` folder and the python file so that should be clear. I've also added the correct labels in the `c.json` file as requested. 

> In Sublime Text parlance your package isn't a theme, it's a (collection of) color scheme(s). So I would suggest using the correct wording here also.

About this, the `Soothing pastel theme for <App Name>` is the description convention we follow at Catppuccin so I don't think we need to remove that?

File: repository/c.json
Comment (txtsd) [inline repository/c.json:425]: This should be `"sublime_text" : ">=3100"` because `.sublime-color-scheme` support was added in ST 3.1

**Review by txtsd — APPROVED** [review]

**Comment by braver** [comment]
Nice 👍🏻

**Pull request merged**
